src/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <ethernetModule.h>
+#include <headerConfig.h>
```

Comment:
```suggestion
```
As mentioned in other comment `headerConfig` is empty and should probably be removed.